website.qmd

```diff
@@ -0,0 +1,9 @@
+### Package website modes
+
+Epiverse-TRACE packages use the [`{pkgdown}` package](https://pkgdown.r-lib.org/) to generate websites. `{pkgdown}` provides [development modes](https://pkgdown.r-lib.org/reference/build_site.html#development-mode) (in the [`_pkgdown.yml` file](https://pkgdown.r-lib.org/articles/customise.html)) to generate different sites for the development and released versions of your package.
+
+When the package is not yet on CRAN, we use the `unreleased` development mode. In this case, we use a visual element to highlight the package development version in red. An example is the [website](https://epiverse-trace.github.io/superspreading/) for `{superspreading}` and its associated `_pkgdown` [configuration](https://github.com/epiverse-trace/superspreading/blob/main/_pkgdown.yml#L12-L13).
+
+After the package is accepted on CRAN, we switch to `auto` mode, which provides two versions of the website: `release` and `devel`. Here, an example is the `{cfr}`package [website](https://epiverse-trace.github.io/cfr/), which is configured [here](https://github.com/epiverse-trace/cfr/blob/main/_pkgdown.yml#L5-L6). We use this configuration for the CRAN packages because we expect most users to install the package from CRAN and so we direct them to the CRAN site associate with the installed release. The additional `devel` option here allows users to manually switch to it if they installed the development version from GitHub.
```

Comment:
> The additional `devel` option here allows users to manually switch to it if they installed the development version from GitHub.

This sentence is unclear to me. I feel it would be more informative to fully explain that the default site under `auto` is `release` but there is a `devel` site also rendered and give the users instructions on how to access the `devel` site. From what I can see on the Epiverse packages that use `auto` there is no direct link to `devel` sites so users need to type `/dev` into the URL, but there may be a better way to do this that I'm unaware of.

Reply:
Does 3d55eb92362c03f1308630dc51d784f196e76181 clarify it?

Reply:
Yes! 👍